docs/en/enterprise-edition/policy-reference/azure-policies/azure-networking-policies/bc-azure-2-50.adoc

```diff
@@ -0,0 +1,53 @@
+
+== Azure Storage Account storing Machine Learning workspace high business impact data is publicly accessible
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/graph_checks/azure/AzureMLWorkspaceHBIPublicNetwork.yaml[CKV2_AZURE_50]
+
+|Severity
+|HIGH
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is checking to make sure that the Azure Storage Account, which stores high business impact data for the Machine Learning workspace, is not publicly accessible. The reason this is important is because it involves sensitive information that could be detrimental to the business if it was accessed by unauthorized individuals. If this data was leaked or misused, it could lead to significant financial and reputational damage. Therefore, the data should be secured and not available to the public to ensure its confidentiality and integrity.
```

Comment:
This policy checks whether Azure Storage Accounts for Machine Learning workspaces are publicly accessible. Public access to these accounts storing sensitive business data poses a significant security risk. Leaks or misuse could lead to financial and reputational damage. By ensuring private access, this policy safeguards data confidentiality and integrity.